src/azul/indexer/document.py

```diff
@@ -604,7 +604,7 @@ class DocumentSource(SourceRef[SimpleSourceName, SourceRef]):
 
     @classmethod
     def from_json(cls, source: JSON) -> 'DocumentSource':
-        return cls(id=source['id'], name=SimpleSourceName(source['name']))
+        return cls(id=source['id'], name=SimpleSourceName(name=source['name']))
```

Comment:
some funk here w/ using `name` so many times
Should be refactored to  `spec=SimpleSourceName(name=source['name'])` 
with changes from https://github.com/DataBiosphere/azul/issues/2843